### DISCUSSION TITLE
ng ant site is down

### DISCUSSION THREAD

**Discussion by ra-jm** [NG-ZORRO/ng-zorro-antd#7780] [General]
For more than couple of hours i couldn't access the documentations, that greatly affects the progress.

https://ng.ant.design/docs/introduce/en

![image](https://user-images.githubusercontent.com/102757408/208312323-1e7c6c51-b54f-448e-87bc-895dc449b025.png)

That being said, in case of downtimes is there any alternative places where I can refer the documents?

**Comment by Axellenomorph** [comment]
Same issue right now

**Reply by kwongtn** [reply]
Similar issue with me too
![image](https://github.com/NG-ZORRO/ng-zorro-antd/assets/5886584/b2e573dd-0e2a-48b9-9cfe-845c34e9932c)

**Reply by tungnxu** [reply]
i got same problems, can not access code snippet example, please fix
